frontend/tsconfig.json

```diff
@@ -1,11 +1,16 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    // TODO: research if es2015 is supported by enough browsers
```

Comment:
Reports https://caniuse.com/?search=es2015 suggest it's supported enough (for ex, chrome's last unsupported version's release was in 2015)

Reply:
Cool! I changed this to avoid some other complications with how TypeScript was compiling stuff, so if only very old browsers don't support it, I don't think we need to be concerned.